### ISSUE TITLE
Add Enum native conversion?

### ISSUE THREAD

**Issue by mlvn23** [uniVocity/univocity-parsers#31]
Can you please add a native generic enum conversion? I think it's one of the most basic data types and it should be included in the library. 

I did try to create my own, but I couldn't make it generically since I can only pass Strings to the @Convert annotation args, so I ended creating one conversion class per enum type, which is verbose. I could have passed the class name in the String, I would suppose.

Maybe you should allow an Object[] args for @Convert annotation instead of just String[]?

**Comment by jbax** [comment]
Support introduced in this commit: 
https://github.com/uniVocity/univocity-parsers/commit/14c4bb587b216a47d3e22c2126e63d37cbde7ed8

I released SNAPHOST version 1.5.5. WIll release final version and merge into 1.6.0 once documentation is ready.

**Comment by jbax** [comment]
Merged into 1.6.0. Please update to 1.6.0-SNAPSHOT
